sky/clouds/service_catalog/gcp_catalog.py

```diff
@@ -16,8 +16,15 @@
 if typing.TYPE_CHECKING:
     from sky.clouds import cloud
 
-_df = common.read_catalog('gcp/vms.csv')
-_image_df = common.read_catalog('gcp/images.csv')
+# Pull the latest catalog every 2 weeks.
+# GCP guarantees that the catalog is updated at most once per 30 days, so
+# we do not need to pull too frequently.
+_PULL_FREQUENCY_HOURS = 24 * 14
```

Comment:
In the worst case the user's catalog will be updated 14 days after the actual change is made, right? Why don't we decrease this a bit (e.g., 1 week or so)? I think that will do more good than harm.

Reply:
Yes, that is a good point. I was afraid that the fetch_gcp is not robust enough, causing our catalog somehow corrupted, but it should be fine to set it to 7 days, and we can fix any issues for that if it ever happens.